src/storageManager.js

```diff
@@ -64,7 +64,9 @@ export function newStorageManager({gvlid, moduleName, moduleType} = {}) {
       if (result && result.valid) {
         const domainPortion = (domain && domain !== '') ? ` ;domain=${encodeURIComponent(domain)}` : '';
         const expiresPortion = (expires && expires !== '') ? ` ;expires=${expires}` : '';
-        document.cookie = `${key}=${encodeURIComponent(value)}${expiresPortion}; path=/${domainPortion}${sameSite ? `; SameSite=${sameSite}` : ''}`;
+        var isNone = (sameSite != null && sameSite.toLowerCase() == 'none')
+        var secure = (isNone) ? '; Secure' : '';
```

Comment:
Can you use const here and for isNone as well.

Reply:
Made requested change to use const. Thanks.